.golangci.yml

```diff
@@ -174,6 +174,10 @@ issues:
         - govet
       text: "lostcancel"
     # Sometimes test functions just get long :(
+    - path: .*
+      linters:
+        - golint
+      text: "returns unexported type"
```

Comment:
I don't quite follow what this is doing... it looks like it may be overly general... but I can't tell.  Also, the comment about it goes with the next exception.

Reply:
oops! Yes, you are right, it is wrong to change. Fixed!